### DISCUSSION TITLE
Add `resize`, `padding` function to pixeldata 为 PixelData 实现 padding、resize 的功能

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#377] [MMEngine]
# Add `resize`, `padding` function to pixeldata 
 为 PixelData 实现 padding、resize 的功能 

Add `resize`, `padding` function to pixeldata [[https://github.com/open-mmlab/mmengine/blob/cfee6d55a7d065d34bc27e3f68652faea99ecbfc/mmengine/structures/pixel_data.py#L130](https://github.com/open-mmlab/mmengine/blob/cfee6d55a7d065d34bc27e3f68652faea99ecbfc/mmengine/structures/pixel_data.py#L130)](https://github.com/open-mmlab/mmengine/blob/cfee6d55a7d065d34bc27e3f68652faea99ecbfc/mmengine/structures/pixel_data.py#L130](https://github.com/open-mmlab/mmengine/blob/cfee6d55a7d065d34bc27e3f68652faea99ecbfc/mmengine/structures/pixel_data.py#L130)) 

为 PixelData 实现 padding、resize 的功能。[[https://github.com/open-mmlab/mmengine/blob/cfee6d55a7d065d34bc27e3f68652faea99ecbfc/mmengine/structures/pixel_data.py#L130](https://github.com/open-mmlab/mmengine/blob/cfee6d55a7d065d34bc27e3f68652faea99ecbfc/mmengine/structures/pixel_data.py#L130)](https://github.com/open-mmlab/mmengine/blob/cfee6d55a7d065d34bc27e3f68652faea99ecbfc/mmengine/structures/pixel_data.py#L130](https://github.com/open-mmlab/mmengine/blob/cfee6d55a7d065d34bc27e3f68652faea99ecbfc/mmengine/structures/pixel_data.py#L130)) 
 

## Task Information

**Repository**：MMEngine

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：14 days

## Task Status

**Claimant's Github ID**：CescMessi

**Claim Time**：2023年10月28日

**PR Link**：https://github.com/open-mmlab/mmengine/pull/1416

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by CescMessi** [comment]
Claim the mission

**Comment by CescMessi** [comment]
PR Link: https://github.com/open-mmlab/mmengine/pull/1416
